Assets/Scripts/ObjectPool/PoolManager.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PoolManager : MonoBehaviour
```

Comment:
Don't think it's necessary and a good idea to create a singleton instance of a poolmanager like this. Classes can hold their own instance of an ObjectPool. This encapsulates the functionality, following the SOLID guidelines